src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -390,7 +390,7 @@ else if (text.isEmpty()) {
     }
 
     int downloadState = messageRecord.getDownloadState();
-    if (downloadState != DcMsg.DC_DOWNLOAD_DONE) {
+    if (downloadState != DcMsg.DC_DOWNLOAD_DONE && downloadState != DC_DOWNLOAD_UNDECIPHERABLE) {
```

Comment:
for me, a positive list is easier to read,  and also matches better the conditions in the branch.

which one is better in the future, is unclear, however - so feel free to ignore this suggestion :)

```suggestion
    if (downloadState == DcMsg.DC_DOWNLOAD_AVAILABLE || downloadState == DcMsg.DC_DOWNLOAD_FAILURE || downloadState == DcMsg.DC_DOWNLOAD_IN_PROGRESS) {
```

Reply:
k, too slow :) but no need to change, just forget my suggestion :)

Reply:
I followed your suggestion in a new PR